### PR TITLE
Manage both detached and attached previews

### PR DESCRIPTION
**Pull request by K-LV**
WIP fix for humphd/brackets#195. This patch should manage the detached and attached previews.

### CODE REVIEW

File: lib/iframe-browser.js [hunk 1]
```diff
@@ -83,6 +83,10 @@ define(function (require, exports, module) {
             var iframe = getBrowserIframe();
             if(iframe) {
                 iframe.src = url;
+                var detachedWindow = getDetachedPreview();
```
Comment (humphd) [inline lib/iframe-browser.js]: Flatten this a bit:

``` js
 if(!url) {
  return;
}

var iframe = getBrowserIframe();
if(iframe) {
  iframe.src = url;
}

var detachedWindow = getDetachedPreview();
if(detachedWindow) {
  detachedWindow.location.replace(url);
}
```

Also, update the comment above this function to reflect that it's more than just the iframe.

**Comment by humphd** [comment]
r+ with my fixes addressed.

File: main.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ define(function (require, exports, module) {
         PostMessageTransport = require("lib/PostMessageTransport"),
         FileSystem           = brackets.getModule("filesystem/FileSystem");
 
-        ExtensionUtils.loadStyleSheet(module, "stylesheets/style.css");
+    ExtensionUtils.loadStyleSheet(module, "stylesheets/style.css");
```
Comment (K-LV) [inline main.js:34]: Fixed jshint error.

**Comment by humphd** [comment]
looks good! r+

**Pull request merged**